output/elastic/outputelastic.go

```diff
@@ -21,6 +21,7 @@ const ModuleName = "elastic"
 type OutputConfig struct {
 	config.OutputConfig
 	URL             []string `json:"url"`               // elastic API entrypoints
+	ResolvedURLs    []string `json:"-"`                 // URLs after resolving environment vars
```

Comment:
The field is generated by code, so it should be a private field (lowercase prefix), except you want it can be configured in yaml.